[2023-06-27T09:09:49.351+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe permission to 509
[2023-06-27T09:09:49.352+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe'
[2023-06-27T09:09:49.381+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe permission to 509
[2023-06-27T09:09:49.382+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe'
[2023-06-27T09:09:49.552+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_pipe manual__2023-06-27T09:09:11.133710+00:00 [queued]>
[2023-06-27T09:09:49.562+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.create_pipe manual__2023-06-27T09:09:11.133710+00:00 [queued]>
[2023-06-27T09:09:49.562+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T09:09:49.578+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_pipe> on 2023-06-27 09:09:11.133710+00:00
[2023-06-27T09:09:49.584+0000] {standard_task_runner.py:57} INFO - Started process 12527 to run task
[2023-06-27T09:09:49.588+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Setup', 'create_pipe', 'manual__2023-06-27T09:09:11.133710+00:00', '--job-id', '201', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpsygfqb3i']
[2023-06-27T09:09:49.591+0000] {standard_task_runner.py:85} INFO - Job 201: Subtask create_pipe
[2023-06-27T09:09:49.636+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe permission to 509
[2023-06-27T09:09:49.637+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T09:09:11.133710+00:00/task_id=create_pipe'
[2023-06-27T09:09:49.639+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Setup.create_pipe manual__2023-06-27T09:09:11.133710+00:00 [running]> on host 4d37da440d5c
[2023-06-27T09:09:49.754+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Setup' AIRFLOW_CTX_TASK_ID='create_pipe' AIRFLOW_CTX_EXECUTION_DATE='2023-06-27T09:09:11.133710+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-27T09:09:11.133710+00:00'
[2023-06-27T09:09:49.756+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T09:09:49.757+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run-operation create_pipe --args \'{                            "pipe_name": "get_new_files",                             "table_name": "ed_pipeline_logs",                             "dry_run": false}\'']
[2023-06-27T09:09:49.768+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T09:09:52.533+0000] {subprocess.py:93} INFO - [0m09:09:52  Running with dbt=1.5.1
[2023-06-27T09:09:59.626+0000] {subprocess.py:93} INFO - [0m09:09:59  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-27T09:09:59.627+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2023-06-27T09:09:59.628+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day
[2023-06-27T09:09:59.628+0000] {subprocess.py:93} INFO - - models.badge_5.create_stage
[2023-06-27T09:09:59.691+0000] {subprocess.py:93} INFO - [0m09:09:59  Found 2 models, 0 tests, 0 snapshots, 0 analyses, 455 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-27T09:10:03.526+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T09:10:03.555+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Setup, task_id=create_pipe, execution_date=20230627T090911, start_date=20230627T090949, end_date=20230627T091003
[2023-06-27T09:10:03.642+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T09:10:03.669+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
